lib/iris/experimental/fieldsfile.py

```diff
@@ -20,12 +20,15 @@
 """
 
 from __future__ import (absolute_import, division, print_function)
+import os
```

Comment:
This import doesn't belong here; it should be in a (new) section between here and the iris ones.